### PR TITLE
fix healthcheck issue for sno/compact with additional workers

### PR DESCRIPTION
**Pull request by jianlinliu**
To fix https://github.com/openshift/release/pull/38689#issuecomment-1527011973

### CODE REVIEW

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-baremetal-3-plus-1-agent-ipv4-static-p2-f14

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-baremetal-3-plus-1-agent-ipv4-static-p2-f14

**Comment by jianlinliu** [comment]
@gpei can you help review here.

File: ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh [hunk 1]
```diff
@@ -163,6 +163,67 @@ function wait_clusteroperators_continous_success() {
...
+        echo "Real-time detected unhealthy mcp:"
+        oc get mcp -o custom-columns=NAME:metadata.name,CONFIG:spec.configuration.name,UPDATING:status.conditions[?\(@.type==\"Updating\"\)].status,DEGRADED:status.conditions[?\(@.type==\"Degraded\"\)].status,DEGRADEDMACHINECOUNT:status.degradedMachineCount | grep -v "False.*False.*0"
+        echo "Real-time full mcp output:"
+        oc get mcp
+        echo ""
+        unhealthy_mcp_names=$(echo "${unhealthy_mcp}" | awk '{print $1}')
+        echo "Using oc describe to check status of unhealthy mcp ..."
+        for mcp_name in ${unhealthy_mcp_names}; do
+          echo "Name: $mcp_name"
+          oc get mcp $mcp_name -o json || echo "oc get mcp -o json failed"
```
Comment (gpei) [inline ci-operator/step-registry/cucushift/installer/check/cluster-health/cucushift-installer-check-cluster-health-commands.sh:190]: As the above prompt is saying "Using oc describe to check status of unhealthy mcp"
so I guess we can keep using `oc describe mcp` here.

I compared the output of `oc describe mcp master` and `oc get mcp master -o json`, seems 
`oc describe` is more useful as it has Events printing out.

**Pull request merged**